src/v0/graphqlFetch.ts

```diff
@@ -60,17 +61,22 @@ export function createGraphqlFetch({
   endpoint = () => 'https://api.manifold.co/graphql',
   version,
   retries = 3,
+  clientId,
 }: CreateGraphqlFetch): GraphqlFetch {
   const options: RequestInit = {
     method: 'POST',
     headers: {
       Connection: 'keep-alive',
       'Content-type': 'application/json',
-      ...(element ? { 'x-mui-component': `${element.tagName}@${version}` } : {}),
+      'x-mui-component': `${element.tagName}@${version}`,
       'x-manifold-mui-core-version': '<@NPM_PACKAGE_VERSION@>',
     },
   };
 
+  if (clientId) {
+    options.headers['Manifold-Client-ID'] = clientId;
+  }
```

Comment:
👍

Reply:
Also dunno if it matters, but James was asking whether or not we should prefix headers with `x-*`. He seemed to think we didn’t need to. I personally don’t care; just wanted to mention it came out of that discussion that the client ID header doesn’t have an `x-*`